llama_stack/templates/meta-reference-gpu/meta_reference.py

```diff
@@ -112,15 +112,15 @@ def get_distribution_template() -> DistributionTemplate:
                 "Port for the Llama Stack distribution server",
             ),
             "INFERENCE_MODEL": (
-                "meta-llama/Llama-3.2-3B-Instruct",
+                "Llama3.2-3B-Instruct",
```

Comment:
why did you change these?

Reply:
We want to use the model_id name as checkpoint path name to read checkpoint from. Changing the name make 
- It aligns with the checkpoint folder name after running 'llama model download'
- It aligns with the descriptor name defined in llama-models SKU list https://github.com/meta-llama/llama-models/blob/bf5b0c4fe74e3b51ed5904ab65e3f671b194d2a9/models/datatypes.py#L95

The original name "meta-llama/Llama-3.2-3B-Instruct" is the hugging face repo name which I feel is a little bit weird here as 'llama model download' is default to download model from meta